mergify_engine/queue/merge_train.py

```diff
@@ -57,6 +57,34 @@
 }
 
 
+class UnexpectedChange:
+    pass
+
+
+@dataclasses.dataclass
+class UnexpectedDraftPullRequestChange(UnexpectedChange):
+    draft_pull_request_number: github_types.GitHubPullRequestNumber
+
+    def __str__(self) -> str:
+        return f"the draft pull request {self.draft_pull_request_number}has been manually updated"
+
+
+@dataclasses.dataclass
+class UnexpectedUpdatedPullRequestChange(UnexpectedChange):
+    updated_pull_request_number: github_types.GitHubPullRequestNumber
+
+    def __str__(self) -> str:
+        return f"the updated pull request #{self.updated_pull_request_number} has been manually updated"
+
+
+@dataclasses.dataclass
+class UnexpectedBaseBranchChange(UnexpectedChange):
+    base_sha: github_types.SHAType
+
+    def __str__(self) -> str:
+        return f"the base branch head has moved manually to {self.base_sha}"
```

Comment:
```suggestion
        return f"an external action moved the branch head {self.base_sha}"
```
Not a fan of this, but I think it sounds better as it really indicates something which is not Mergify did something wrong.
Better idea?